### DISCUSSION TITLE
Feature Idea: Turn off Magsafe LED when sleeping

### DISCUSSION THREAD

**Discussion by andrej-kocijan** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1212] [Ideas]
No idea if this is possible technically (but probably should be considering sleep does not override the always off setting), but I would love if "Stop charging when sleeping" could also cause the Magsafe LED to turn off.

My reasoning: I like the magsafe LED in general when I am using my mac, but find it redundant when I am away from it, letting it sleep.

**Comment by MatthiasKerbl** [comment]
Hi @andrej-kocijan,

Great idea and thank you for your input. We will consider it in our future development.

Best,
Matthias
